src/main/features/claim/views/check-and-send.njk

```diff
@@ -14,48 +14,58 @@
 
     {{ tableStart("yourDetails", "Your details") }}
     {{
-      partyDetailsSummaryFragment(
-        partyDetails = draftClaim.claimant.partyDetails
-      )
+        partyDetailsSummaryFragment(
+          partyDetails = draftClaim.claimant.partyDetails,
+          changeLinks = {
+            name: paths.claimantIndividualDetailsPage.uri,
+            address: paths.claimantIndividualDetailsPage.uri,
+            dateOfBirth: paths.claimantDateOfBirthPage.uri
+          }
+        )
     }}
-    {{ row("Phone:", draftClaim.claimant.mobilePhone.number) }}
+    {{ row("Phone:", draftClaim.claimant.mobilePhone.number, paths.claimantMobilePage.uri) }}
     {{ tableEnd() }}
 
     {{ tableStart("theirDetails", "Their details") }}
-    {{
-      partyDetailsSummaryFragment(
-        partyDetails = draftClaim.defendant.partyDetails,
-        showCorrespondenceAddressSection = false
-      )
+      {{
+          partyDetailsSummaryFragment(
+            partyDetails = draftClaim.defendant.partyDetails,
+            showCorrespondenceAddressSection = false,
+              changeLinks = {
+              name: paths.defendantIndividualDetailsPage.uri,
+              address: paths.defendantIndividualDetailsPage.uri
+           }
+        )
     }}
-    {{ row("Email:", draftClaim.defendant.email.address) }}
+
+    {{ row("Email:", draftClaim.defendant.email.address , paths.defendantEmailPage.uri) }}
```

Comment:
and here

Reply:
Updated

Reply:
All spaces have been updated with your suggested changes